### PR TITLE
BAM-2445: Removed some references to BAMX_EVENT_DATA and user_email

### PR DESCRIPTION
**Pull request by Abalaxy**
References to window.BAMX_EVENT_DATA in the **Adding Information for the Page View Tag** section were causing confusion, so these were removed. 

Also, references to `user_email` being stored were removed, and `product_brand` was added to the **Adding Information About Your Checkout Page** section, both in the table and snippet.

### CODE REVIEW

File: docs/tagmerchant.rst [hunk 1]
```diff
@@ -172,10 +155,6 @@ information into our data layer using Javascript before loading the Narrativ tag
 
   <!-- begin NARRATIV jstag -->
   <script type="text/javascript">
-      window.BAMX_EVENT_DATA = {
-          user_email: {{UserEmail}},
-          user_id: {{UserID}}
-      }
       (function (account) {
```
[the remark this answers is not in the document]
Comment (Abalaxy) [reply docs/tagmerchant.rst]: Where do you want the new lines?

File: docs/tagmerchant.rst [hunk 2]
```diff
@@ -172,10 +155,6 @@ information into our data layer using Javascript before loading the Narrativ tag
 
   <!-- begin NARRATIV jstag -->
```
[the remark this answers is not in the document]
Comment (Abalaxy) [reply docs/tagmerchant.rst]: So `<!-- begin NARRATIV jstag -->` should be `<!-- begin Narrativ jstag -->`, and so on?

File: docs/tagmerchant.rst [hunk 3]
```diff
@@ -4,9 +4,10 @@ Narrativ Advertiser Tag
 Functionality
 -------------
 
-The Narrativ advertiser tag allows Narrativ to track user behavior on an advertiser's site. Our auction system optimizes advertiser acquisition and ROI by analyzing data on page views and purchase behavior. Any other PII that is
-captured, such as names, email addresses, etc. will be hashed and never stored in plain text. The tag runs
-asynchronously in the background so there is no impact to page load times.
+The Narrativ advertiser tag allows Narrativ to track user behavior on an advertiser's site. Our auction system 
+optimizes advertiser acquisition and ROI by analyzing data on page views and purchase behavior. Any other Personal 
+Identifying Information (PII) that is captured, such as names, etc. will be hashed and never stored in plain text. The
```
[the remark this answers is not in the document]
Comment (PhilWinchester) [reply docs/tagmerchant.rst]: What do you think @c-nichols @pbecotte?

[the remark this answers is not in the document]
Comment (pbecotte) [reply docs/tagmerchant.rst]: Once name and email have been removed from the docs, we should just say "No PII is captured"

[the remark this answers is not in the document]
Comment (c-nichols) [reply docs/tagmerchant.rst]: ^

[the remark this answers is not in the document]
Comment (Abalaxy) [reply docs/tagmerchant.rst]: I made this change, let me know if you'd like any further edits.

**Pull request merged**